Apps/WebClient/src/ClientApp/app/views/timeline.vue

```diff
@@ -8,47 +8,57 @@
     <h1 id="subject">
       Health Care Timeline: TODO
     </h1>
+    <ul id="example-1">
+      <li v-for="medication in prescriptions">
+        Generic Name: {{ medication.genericName }}
+      </li>
+    </ul>
   </div>
 </template>
 
 <script lang="ts">
 import Vue from "vue";
 import { Component } from "vue-property-decorator";
 import { State, Action, Getter } from "vuex-class";
+import { IMedicationService } from "@/services/interfaces";
 import LoadingComponent from "@/components/loading.vue";
 import container from "@/inversify.config";
 import SERVICE_IDENTIFIER from "@/constants/serviceIdentifiers";
+import User from "@/models/user";
+import Prescription from "@/models/prescription";
+
+const namespace: string = "user";
 
 @Component({
   components: {
     LoadingComponent
   }
 })
 export default class TimelineComponent extends Vue {
-  //private items: ImmsData[] = [];
+  @Getter("user", { namespace }) user: User;
+  private prescriptions: Prescription[] = [];
   private isLoading: boolean = false;
   private hasErrors: boolean = false;
   private sortyBy: string = "date";
   private sortDesc: boolean = false;
 
   mounted() {
     this.isLoading = true;
-    // TODO: add the medication service
-    /*const immsService: IImmsService = container.get(
-      SERVICE_IDENTIFIER.ImmsService
-    );*/
+    const medicationService: IMedicationService = container.get(
+      SERVICE_IDENTIFIER.MedicationService
+    );
 
-    /*immsService
-      .getItems()
+    medicationService
+      .getPatientPrescriptions(this.user.hdid)
```

Comment:
I suggest we rename this to getPatientMedications() since prescriptions will be a different concept when we get there.